test/client/api.test.js

```diff
@@ -0,0 +1,139 @@
+/* @flow */
+import { describe, beforeEach, it, expect, vi } from "vitest";
+
+import { getCurrentScript, request, memoize } from "@krakenjs/belter/src";
+import { createAccessToken, createOrder } from "../../src/api";
+
+vi.mock("@krakenjs/belter/src", async () => {
+  const actual = await vi.importActual("@krakenjs/belter/src");
+  return {
+    ...actual,
+    getCurrentScript: vi.fn(),
+    request: vi.fn(),
+  };
+});
+
+describe("api cases", () => {
+  let order;
+  const invalidClientId = "invalid-client-id";
+  const emptyResponseClientId = "empty-response-client-id";
+  const createOrderValidId = "create-order-valid-order-id";
+  const expectedToken =
+    "A21AAKNZBaqilFBC4dVVz-tr-ySIT78NREeBidy3lkGdr-EA8wbhGrByPayhgnJRPE5xg4QW46moDbCFjZ13i1GH-Ax4SjtjA";
+  const defaultAuthResponse = {
+    scope: "https://uri.paypal.com/services/invoicing",
+    access_token: expectedToken,
+    token_type: "Bearer",
+    app_id: "APP-80W284485P519543T",
+    expires_in: 31838,
+    nonce: "2022-03-07T22:41:38ZqHkiC0_odfzFwo27_X0wVuF67STYq39KRplBeeyY2bk",
+    error: null,
+  };
+
+  beforeEach(() => {
+    memoize.clear();
+    window.__PAYPAL_DOMAIN__ = "testurl";
+    getCurrentScript.mockReturnValue({
```

Comment:
TODO: there was a lot of futzing with replacing the `insertMockSDKScript` or whatever it was. But now that I've gone through all this, I want to circle back and refresh on how to make a simple util. It could just be at the vitest setup level if we wanted, too.